framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -206,26 +233,21 @@ public void viewpointAdaptMethod(
      * @param newClassTree invocation of the constructor
      * @param constructorType type of the constructor; is side-effected by this method
      */
-    public void viewpointAdaptConstructor(
+    public void atConstructorInvocation(
             NewClassTree newClassTree, AnnotatedExecutableType constructorType) {
         if (!hasDependentAnnotations()) {
             return;
         }
-        List<? extends ExpressionTree> args = newClassTree.getArguments();
-        viewpointAdaptExecutable(newClassTree, constructorType, args);
+        atInvocation(newClassTree, constructorType);
     }
 
     /**
      * Viewpoint-adapts a method or constructor invocation.
      *
      * @param tree invocation of the method or constructor
      * @param methodType type of the method or constructor; is side-effected by this method
```

Comment:
Throughout, please clarify whether this is the declared type.  I think it's a copy of the declared type.